data/859/767/13/85976713.geojson

```diff
@@ -2,251 +2,160 @@
-    "ne:MAX_BBYMIN":34.95,
-    "ne:MAX_NATSCA":50.0,
-    "ne:MAX_PERKM":305.0,
-    "ne:MAX_PERMI":189.0,
-    "ne:MAX_POP10":657215,
-    "ne:MAX_POP20":657215,
-    "ne:MAX_POP300":0,
-    "ne:MAX_POP310":0,
-    "ne:MAX_POP50":657215,
-    "ne:MEAN_BBXC":-106.630354,
-    "ne:MEAN_BBYC":35.136985,
-    "ne:MEGACITY":0,
-    "ne:MEGANAME":null,
-    "ne:MIN_AREAKM":522.0,
-    "ne:MIN_AREAMI":202.0,
-    "ne:MIN_BBXMAX":-106.475,
-    "ne:MIN_BBXMIN":-106.775,
-    "ne:MIN_BBYMAX":35.316667,
-    "ne:MIN_BBYMIN":34.95,
-    "ne:MIN_PERKM":305.0,
-    "ne:MIN_PERMI":189.0,
-    "ne:NAME":"Albuquerque",
-    "ne:NAMEALT":null,
-    "ne:NAMEASCII":"Albuquerque",
-    "ne:NAMEDIFF":0,
-    "ne:NAMEPAR":null,
-    "ne:NATSCALE":50,
-    "ne:NOTE":null,
-    "ne:POP1950":0,
-    "ne:POP1955":0,
-    "ne:POP1960":0,
-    "ne:POP1965":0,
-    "ne:POP1970":0,
-    "ne:POP1975":0,
-    "ne:POP1980":0,
-    "ne:POP1985":0,
-    "ne:POP1990":0,
-    "ne:POP1995":0,
-    "ne:POP2000":0,
-    "ne:POP2005":0,
-    "ne:POP2010":0,
-    "ne:POP2015":0,
-    "ne:POP2020":0,
-    "ne:POP2025":0,
-    "ne:POP2050":0,
-    "ne:POP_MAX":898642,
-    "ne:POP_MIN":552804,
-    "ne:POP_OTHER":651927,
-    "ne:RANK_MAX":11,
-    "ne:RANK_MIN":11,
-    "ne:SCALERANK":4,
-    "ne:SOV0NAME":"United States",
-    "ne:SOV_A3":"USA",
-    "ne:TIMEZONE":"America/Denver",
-    "ne:UN_ADM0":null,
-    "ne:UN_FID":0,
-    "ne:UN_LAT":0.0,
-    "ne:UN_LONG":0.0,
-    "ne:WORLDCITY":0,
-    "qs:a0":"United States",
-    "qs:a1":"*New Mexico",
-    "qs:a1_lc":"35",
-    "qs:a1r":"*",
-    "qs:adm0":"United States",
-    "qs:id":0,
-    "qs:la_lc":"*",
-    "qs:level":"locality",
-    "qs:loc":"Los Ranchos de Albuquerque",
-    "qs:loc_alt":"Los Ranchos de Albuquerque village",
-    "qs:loc_lc":"3543930",
-    "qs:pop":0,
-    "qs:source":"AUS Census",
-    "qs:type":"G4110",
-    "src:geom":"quattroshapes",
-    "src:geom_alt":[
-        "quattroshapes_pg"
-    ],
-    "src:lbl_centroid":"mapshaper",
-    "src:population":"geonames",
-    "wd:wordcount":1515,
-    "wof:belongsto":[
-        102191575,
-        85633793,
-        85688493,
-        102081199
-    ],
-    "wof:breaches":[],
-    "wof:concordances":{
-        "fips:code":"3543930",
-        "gn:id":5476960,
-        "gp:id":2442149,
-        "qs_pg:id":1092715,
-        "uscensus:geoid":"3543930",
-        "wd:id":"Q1933179",
-        "wk:page":"Los Ranchos de Albuquerque, New Mexico"
+    "edtf:cessation": "",
+    "edtf:inception": "",
+    "geom:area": 0.0011154685865000524,
+    "geom:area_square_m": 11276015.16875,
+    "geom:bbox": "-106.678236,35.136192,-106.621803,35.189493",
+    "geom:latitude": 35.162892612304354,
+    "geom:longitude": -106.64825604551109,
+    "gn:elevation": 1520,
+    "gn:latitude": 35.16199,
+    "gn:longitude": -106.6428,
+    "gn:population": 6024,
+    "iso:country": "US",
+    "lbl:latitude": 35.161791,
+    "lbl:longitude": -106.645551,
+    "lbl:max_zoom": 15,
```

Comment:
Is there an issue with these values becoming ints, rather than floats?

Reply:
We had an earlier question about this and determined the Go tooling prefers ints where there aren't significant decimal values, else it'll keep the float. Which is _weird_ but we live within the narrative of various languages and toolings.